cmd/ipfs/goreq.go

```diff
@@ -0,0 +1,3 @@
+// +build !go1.5		
+
+`IPFS needs to be built with go version 1.5 or greater`
```

Comment:
maybe once go1.5 is more widely adopted we can remove this again? I had thought that we were able to fail the builds by using the `for range` syntax, but apparently that compiles under 1.4 for some people...